.github/CONTRIBUTING.md

```diff
@@ -6,6 +6,88 @@ Please read EasyPost's general [contribution guidelines](https://github.com/Easy
 
 Please read EasyPost's [code of conduct](https://github.com/EasyPost/.github/blob/main/CODE_OF_CONDUCT.md).
 
+## Semantic versioning
+
+Easy UI follows semantic versioning. We release [patch versions for bug fixes](https://github.com/EasyPost/easy-ui/blob/.github/CONTRIBUTING.md#patch), [minor versions for new features](https://github.com/EasyPost/easy-ui/blob/.github/CONTRIBUTING.md#minor), and [major versions for breaking changes](https://github.com/EasyPost/easy-ui/blob/.github/CONTRIBUTING.md#major). When we make breaking changes, we try to introduce deprecation warnings in a minor version along with the upgrade path so that our users learn about the upcoming changes and migrate their code in advance.
+
+The following sections detail what kinds of changes result in each of major, minor, and patch version bumps:
+
+### Major
+
+- Removal of a component
+- Removal of a prop from a component
+- Change to the type accepted for a prop
+- Breaking changes to minimum version of dependencies
+- Breaking changes to public variables, functions and mixins
+
+### Minor
+
+- New component
+- New prop for a component
+- Additional type accepted for a prop
+- Deprecation of a component, prop, public variable, function, or mixin (ahead of its likely removal in the next major version)
+
+### Patch
+
+- Breaking change to the HTML generated by a component, including addition, removal, or renaming of classes
+- Changes that do not impact public APIs
+- Non-breaking changes to minimum version of dependencies
+- Breaking changes to private variables, functions, and mixins
+
+## Your first pull request
+
+Easy UI is a public project so pull requests are welcome. Before working on a large change, it is best to open an issue first to discuss it with the core team.
+
+### Sending a pull request
+
+We'll review your pull request and either merge it, request changes to it, or close it with an explanation. We'll do our best to provide updates and feedback throughout the process.
+
+**Before submitting a pull request**, please:
+
+1. Review [best practices](https://github.com/EasyPost/easy-ui/blob/main/.github/CONTRIBUTING.md#component-api-best-practices)
+1. Fork the repository and create your branch from `main`
+1. Run `npm install` in the repository root
+1. If you’ve fixed a bug or added code, make sure to add tests
+1. Ensure the test suite passes with `npm test`
+1. Format your code with `npm run format`
+1. Make sure your code lints with `npm run lint`
+1. Create a changeset by running `npm run changes:add`. [More info](https://github.com/EasyPost/easy-ui/blob/main/.github/CONTRIBUTING.md#adding-a-changeset).
```

Comment:
`nit` i think intended to be 2,3,4,5..etc

Reply:
the flavor of markdown on Github turns `1. 1. 1. ...` into `1. 2. 3. ...` so you don't have to keep track of the numbers